cadquery/occ_impl/shapes.py

```diff
@@ -846,6 +887,34 @@ def makeCircle(cls, radius, center, normal):
         w = cls.assembleEdges([circle_edge])
         return w
 
+    @classmethod
+    def makeEllipse(cls, x_radius, y_radius, center, normal, angle1=360.0, angle2=360.0, 
```

Comment:
What is the rationale for having two makeEllipse functions?

Reply:
This is just like makeCircle, it exists in class Edge and Wire.
I didn't fully understand, but again, wanted to keep the ellipse implementation as similar to the circle implementation as possible

Reply:
Indeed, you are right.